configuration/configuration_reference.md

```diff
@@ -562,6 +599,85 @@ Two users would be administrators, they are **Jez** and **lqiao**.
 </view>
 ```
 
+
+## &lt;config-repos&gt; {#config-repos}
+
+The `<config-repos>` element is a container of many `<config-repo>`.
+
+### Example
+
+```xml
+<cruise>
+  ...
+  <config-repos>
```

Comment:
The XML config repo thing is not going to be public. It's not a real plugin. It has been removed from the original doc as well (the google drive one).